addons/sale_project/models/project_project.py

```diff
@@ -405,13 +439,22 @@ def _get_sale_items_domain(self, additional_domain=None):
             domain = expression.AND([domain, additional_domain])
         return domain
 
-    def _get_sale_items(self, with_action=True):
-        domain = self._get_sale_items_domain()
+    def _get_items_id_per_section_id(self):
         return {
-            'total': self.env['sale.order.line'].sudo().search_count(domain),
-            'data': self.get_sale_items_data(domain, limit=5, with_action=with_action),
+            'materials': [],
+            'service_revenues': [],
         }
 
+    def _get_domain_from_section_id(self, section_id):
+        if section_id == 'materials':
+            return self._get_sale_items_domain([('product_id.type', '!=', 'service')])
+        #  When the sale_timesheet module is not installed, all service products are grouped under the 'service revenues' section.
+        return self._get_sale_items_domain([('product_id.type', '=', 'service')])
+
+    def _get_sale_items(self, with_action=True):
+        domain = self._get_sale_items_domain()
+        return self.get_sale_items_data(domain, limit=5, with_action=with_action)
```

Comment:
I think that method could be removed since we now just call it in `get_panel_data`. 🙂